recipe/meta.yaml

```diff
@@ -44,7 +44,6 @@ requirements:
     - libuuid  # [linux]
     - libwebp-base
     - libxml2
-    - m2w64-xz  # [win]
```

Comment:
I think this is fine. My guess is that this was used only to unpack the sdist but it is not needed for a while.